src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/ContractGrantReportingCode.xml

```diff
@@ -1,168 +1,128 @@
-
-  <bean id="ContractGrantReportingCode-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition">
-    <property name="title" value="CG Reporting Code Inquiry"/>
-    <property name="inquirySections">
-      <list>
-        <bean parent="InquirySectionDefinition">
-          <property name="title" value=""/>
-          <property name="numberOfColumns" value="1"/>
-          <property name="inquiryFields">
+    
+    <bean id="ContractGrantReportingCode-sectionDefinition" parent="ContractGrantReportingCode-sectionDefinition-parentBean"/>     
+    <bean id="ContractGrantReportingCode-sectionDefinition-parentBean" abstract="true" parent="sectionDefinition" p:title="">
+        <property name="fields">
             <list>
-              <bean parent="FieldDefinition" p:attributeName="chartOfAccountsCode"/>
-              <bean parent="FieldDefinition" p:attributeName="code"/>
-              <bean parent="FieldDefinition" p:attributeName="name"/>
-              <bean parent="FieldDefinition" p:attributeName="active"/>
+                <bean parent="ContractGrantReportingCode-chartOfAccountsCode"/>
+                <bean parent="ContractGrantReportingCode-code"/>
+                <bean parent="ContractGrantReportingCode-name"/>
+                <bean parent="ContractGrantReportingCode-active"/>
             </list>
-          </property>
-        </bean>
-      </list>
-    </property>
-  </bean>
-
-<!-- Business Object Lookup Definition -->
-
-
-  <bean id="ContractGrantReportingCode-lookupDefinition" parent="ContractGrantReportingCode-lookupDefinition-parentBean"/>
+         </property>
+     </bean>
+     
+    <bean id="ContractGrantReportingCode-inquiryDefinition" parent="ContractGrantReportingCode-inquiryDefinition-parentBean"/>
+    <bean id="ContractGrantReportingCode-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition"
+          p:title="CG Reporting Code Inquiry">
+        <property name="sections">
+            <list>
+                <ref bean="ContractGrantReportingCode-sectionDefinition"/>
+            </list>
+        </property>
+    </bean>
+     
+     
+    <!-- Business Object Lookup Definition -->
 
-  <bean id="ContractGrantReportingCode-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    <property name="title" value="CG Reporting Code Lookup"/>
-    
-    <property name="defaultSort">
-      <bean parent="SortDefinition">
-        <property name="attributeNames">
-          <list>
-            <value>chartOfAccountsCode</value>
-            <value>code</value>
-          </list>
+    <bean id="ContractGrantReportingCode-lookupDefinition" parent="ContractGrantReportingCode-lookupDefinition-parentBean"/>
+    <bean id="ContractGrantReportingCode-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition"
+          p:title="CG Reporting Code Lookup">
+        <property name="defaultSort">
+            <bean parent="SortDefinition" p:sortAscending="true">

```

Comment:
Another lookup whose default sort was previously configured for descending order but is not configured for ascending order.